blink example does not compile

Hello,
new in rust.
Following the readme of the project, i managed to set up the toml to make it work:

```
[package]
name = "xxxx"
version = "0.1.0"
authors = ["xxxx"]
edition = "2018"

[profile.release]
# optimize for size ('z' would optimize even more)
opt-level = 's'
# link with link time optimization (lto).
lto = true
# enable debugging in release mode.
debug = true

[dependencies]
cortex-m = "0.6.0"
nb = "0.1.2"
cortex-m-rt = "0.6.8"
stm32f1 = "0.7.1"

[dependencies.stm32f1xx-hal]
version = "0.2.0"
features = ["stm32f100", "rt"]

```

using 0.2.0 as requested from the readme return the error:

```
error[E0609]: no field `pc13` on type `stm32f1xx_hal::gpio::gpioc::Parts`
```

then i notice 0.3.0 is available, and with that one i get warning about digital::v1 (I read a little bit the issue by my skill level are 0 and i could not understand nothing).

Please please please fix your own doc and warning,
sincerely,

a frustrated beginner

--- a/README.md
+++ b/README.md
@@ -49,7 +49,7 @@ be specified as part of the `Cargo.toml` definition.
 
 ```toml
 [dependencies.stm32f1xx-hal]
-version = "0.2.0"
+version = "0.3.0"
 features = ["stm32f100", "rt"]
 ```
 

